src/main/java/train/common/core/handlers/ItemHandler.java

```diff
@@ -45,6 +42,11 @@ public static boolean handleItems(Entity entity, ItemStack itemstack) {
 	}
 
 	public static boolean handleFreight(Entity entity, ItemStack itemstack) {
+        int logWood = OreDictionary.getOreID("logWood");
```

Comment:
Please check your indentation please, before you use tab character but in this change, you use spaces. It isn't consistent